plugin/src/main/java/com/denizenscript/denizen/objects/CuboidTag.java

```diff
@@ -1547,6 +1661,12 @@ public void adjust(Mechanism mechanism) {
         // <CuboidTag.get>
         // <CuboidTag.add_member[<cuboid>]>
         // <CuboidTag.add_member[<cuboid>].at[<#>]>
+        // @example
+        // # Adds "my_second_cuboid" as a member to "my_cuboid".
+        // - adjust <cuboid[my_cuboid]> add_member:my_second_cuboid
```

Comment:
For all these member-related examples: again, add example data. What does the cuboid look like before/after this operation?